Zephyr.swift

```diff
@@ -91,6 +98,11 @@ public class Zephyr: NSObject {
     override init() {
         super.init()
         NSNotificationCenter.defaultCenter().addObserver(self, selector: "keysDidChangeOnCloud:", name: NSUbiquitousKeyValueStoreDidChangeExternallyNotification, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: "willEnterForeground:", name:
+            UIApplicationWillEnterForegroundNotification, object: nil)
+
+        let ubiquitousStore = NSUbiquitousKeyValueStore.defaultStore()
+        ubiquitousStore.synchronize()
```

Comment:
Do the `synchronize` on one line, as there's no need to store it in a variable if we're only going to call it once.